### PR TITLE
Rename monitor.lua module to statsd.monitor.lua

### PR DESCRIPTION
**Pull request by fmejia97**
**What this PR does / why we need it**:
This PR renames our **monitor.lua** module to **statsd.monitor.lua** Why? Upstream NGINX has a new **monitor.lua** module and we want to stay consistent with upstream.  In order to rebase with upstream we need to rename our lua module.

**Which issue this PR fixes** 
fixes #59 

**Special notes for your reviewer**:

### CODE REVIEW

File: rootfs/etc/nginx/template/nginx.tmpl [hunk 1]
```diff
@@ -67,11 +67,11 @@ http {
           balancer = res
         end
 
-        ok, res = pcall(require, "monitor")
+        ok, res = pcall(require, "statsd.monitor")
```
Comment (ElvinEfendi) [inline rootfs/etc/nginx/template/nginx.tmpl]: can you rename it to `statsd_monitor.lua`?

**Comment by ElvinEfendi** [comment]
please reference the issue this is fixing

File: rootfs/etc/nginx/template/nginx.tmpl (hunk 1 above)
Comment (andrewlouis93) [reply rootfs/etc/nginx/template/nginx.tmpl]: 👍 from what I've seen "statsd.monitor" may mean that the file is  `statsd/monitor.lua`. Could top hat to verify that there's no problem requiring the module - but renaming to `statsd_monitor.lua` should fix that concern.

**Comment by fmejia97** [comment]
@ElvinEfendi Addressed your comments :)

**Comment by ElvinEfendi** [comment]
fixes https://github.com/Shopify/edgescale/issues/554

**Pull request merged**